### PR TITLE
RHDEVDOCS 5870 Release Notes for Pipelines 1.11.3

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.11, pipelines-docs-1.12, pipelines-docs-1.13

Issue:

RHDEVDOCS 5870

Link to docs preview:

https://70408--docspreview.netlify.app/openshift-pipelines/latest/about/op-release-notes#release-notes-1-11-3_op-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/op-release-notes-1-11.adoc [hunk 1]
```diff
@@ -275,3 +275,13 @@ If you installed the `tkn` command line tool using RPM on {op-system-base-full},
...
+
+[id="release-notes-1-11-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.3
+
+With this update, {pipelines-title} General Availability (GA) 1.11.3 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-11-3_{context}"]
+=== Fixed issues
+
+* Before this update, if the final task of a pipeline has failed or wws skipped, {pipelines-shortname} reported validation errors. With this update, a pipeline can succeed even if its final task fails or is skipped.
```
Comment (jkandasa) [inline modules/op-release-notes-1-11.adoc:287]: @mramendi typo,
* wws skipped => was skipped

File: modules/op-release-notes-1-11.adoc [hunk 2]
```diff
@@ -275,3 +275,13 @@ If you installed the `tkn` command line tool using RPM on {op-system-base-full},
 == Known issues
 
 * If you use the Bundles resolver to define a pipeline run and then use the `tkn pac resolve --v1beta1` command for this pipeline run, the command generates incorrect YAML output. The `kind` parameter for the bundle is set to `Task` in the YAML output. As a workaround, you can set the correct value in the YAML data manually. Alternatively, you can use the `opc pac resolve --v1beta1` command or use the version of the `tkn` tool included with {pipelines-shortname} version 1.12.0 or later.
+
+[id="release-notes-1-11-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.3
+
+With this update, {pipelines-title} General Availability (GA) 1.11.3 is available on {OCP} 4.12 and later versions.
```
Comment (ppitonak) [inline modules/op-release-notes-1-11.adoc:282]: We added 4.11 in this release

File: modules/op-release-notes-1-11.adoc (hunk 1 above)
Comment (ppitonak) [reply modules/op-release-notes-1-11.adoc:287]: IMHO the issue description is not accurate. It was failing when pipeline was re-publishing a result of a failed/skipped task.

reproducer in https://issues.redhat.com/browse/SRVKP-3975?focusedId=23797650&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23797650

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: modules/op-release-notes-1-11.adoc (hunk 1 above)
Comment (ppitonak) [reply modules/op-release-notes-1-11.adoc:287]: Actually, I was wrong, ignore previous comment.

**Review by ppitonak — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]

**Review by concaf — APPROVED** [review]

**Review by jkandasa — APPROVED** [review]

**Comment by shipsing** [comment]
@mramendi, The content changes look fine.  Since i am new to the peer review squad, i would ask @bburt-rh to give my review the go-ahead. Please wait until Brain applies the peer-review-done label.

**Comment by jkandasa** [comment]
@shipsing can you please merge this as soon as possible?
1.11.3 release can happen anytime soon.

**Review by sheriff-rh — APPROVED** [review]
Changes look good to me.

**Comment by Srivaralakshmi** [comment]
Thanks @sheriff-rh for the quick and timely help. Appreciate it!

**Pull request merged**